misc-scripts/xref_mapping/XrefParser/HGNCParser.pm

```diff
@@ -70,7 +70,9 @@ use strict;
 use warnings;
 use Carp;
 use Text::CSV;
-use utf8;
+#use utf8;
```

Comment:
Please remove the comments. If we need to remember, this is always there in the git history.